lora-phy/src/sx1276_7_8_9/mod.rs

```diff
@@ -55,14 +55,9 @@ where
     }
 
     // Utility functions
-    async fn write_register(
-        &mut self,
-        register: Register,
-        value: u8,
-        is_sleep_command: bool,
-    ) -> Result<(), RadioError> {
+    async fn write_register(&mut self, register: Register, value: u8) -> Result<(), RadioError> {
```

Comment:
are you using rustfmt? I don't really care how it looks as long as we don't go back and forth every commit

Reply:
Well, after dropping `is_sleep_command: bool,` this fits nicely on single line...

Reply:
yeah I noticed that 1 second after commenting, sry